package.json

```diff
@@ -72,7 +72,7 @@
     "mapbox-gl": ">=0.47.0"
   },
   "engines": {
-    "node": "^12"
+    "node": ">=12"
```

Comment:
I think the use of `>=` in `engines` or in `dependencies` and `devDependencies` should be a no-no. It has caused problems for us in the past because it's far too open ended. From my understanding, specifying this value on dependencies tells npm what version of node is required when it is trying to install this dependency in a given project.

Imagine a scenario where you're working on a project whose own source code works with node 14 and try to install this project with this change implemented. NPM is going to let you because node 14 is >=12, but do we know for sure that this project really works with _all versions of node >=12?_ No, because new versions will continue to come out and something will break eventually. This value should always be something deterministic and bounded on both ends. If you want to publish a version of this dependency that you are sure compiles on Node 12 and 14, so that you can use that version on Node 12 and 14 "client" project, then something like `"node": "^12" | "^14"` would be safer.

Reply:
I made the change, merged, and published a [v0.4.0](https://www.npmjs.com/package/ember-mapbox-composer/v/0.4.0) of this package. That version allows node 12 and 14 so it should work for our purposes here. 

I appreciate you working through feedback like this. I know best practices aren't very well documented here but I'm not expecting you to know this sort of thing ahead of time. It's what code reviews are for, after all! In the future, avoid that syntax when you can. It's probably less dangerous for `engines` but it has caused headaches for me in dependencies because it can cause shifts in installed major versions over time without a developer ever actually touching package.json.